README.md

```diff
@@ -1,172 +1,124 @@
+This section will go over the necessary steps to train your own
+models.
 
-We love Python! To make our lives easier and write good code we follow a couple
-of common best practices:
+## Data
 
-* For local development we recommend using Python's virtual environments. This
-  way you can keep a separate environment for each project without polluting
-  your operating system with additional Python packages. We also recommend using
-  `pip` for managing those packages.
+The first thing that it is required to train neural networks is
+obviously data. If you have your own dataset, great!! You only need to
+make sure that it obeys the following structure:
 
-  To create a new virtual environment (only once):
-
-  ```bash
-  python3 -m venv .env
-  ```
-
-  To activate your virtual environment (every time you want to work on the
-  current project):
-
-  ```bash
-  source .env/bin/activate 
-  ```
-
-  It's always a good practice to upgrade `pip` when you create the virtual environment, before installing other packages. In the past we've noticed some
-  bugs with older versions of `pip`, causing the installation of incompatible
-  versions of several packages.
-
-  ```bash
-  pip install --upgrade pip
-  ```
-
-  To install the project's required packages (whenever dependencies must be
-  updated):
-
-  ```bash
-  pip install -r requirements.txt
-  ```
-
-* We follow
-  [Google's style guide](https://google.github.io/styleguide/pyguide.html).
-  Specifically, we format code with `yapf`, and check for bugs and style
-  problems with `pylint`.
+```
+.
+└── data_set/
+    ├── sim_1/
+    │   └── sim.vtk
+    └── sim_2/
+        └── sim.vtk
+```
 
-  Usually, there's no need to run these tools manually. We added continuous
-  integration (CI) through GitHub, so every time your pull request has errors
-  you'll be alerted via email. Over time you'll get used to these rules and
-  coding consistently will become second nature.
+The folder and file names can change.
 
-  However, if you want to run `yapf` and `pylint` locally, simply install them
-  via `pip`:
+If you do not have your own data you can download one of our datasets
+[here](https://drive.google.com/u/0/uc?id=1T5wXz-igXjrztLt6xgb4CZx4tGwXK8DX&export=download). This
+particular dataset has 1000 examples of objects:
 
-  ```bash
-  pip install yapf pylint
-  ```
+![alt_image](readme_images/dataset_example.png)
 
-* We write tests using `pytest`. See [`example_test.py`](example_test.py) for an
-  example and delete if example after you copied the template to your new
-  project.
+Actually, in the image, we can see an example of an object and the
+predictions of one of our models.
 
-  Tests are also run automatically in GitHub via continuous integration. If you
-  want to run them locally, install `pytest`:
 
-  ```bash
-  pip install pytest
-  ```
+The next step is to pre-process the dataset. To do this just run the
+script:
 
-  and run:
+```bash
+pytohn process --data_dir=<Path to your dataset>
```

Comment:
typo *python

Reply:
This can be different at times... In my case it will be python3

Reply:
> This can be different at times... In my case it will be python3

Even when you create the virtual env? I need to create the virtual env with `python3` but after that it  is just `python`

Reply:
I mean that it depends of the people using. But leave it like this and next I will test this approach and check :)